static/js/common/upload-addon.js

```diff
@@ -261,13 +262,99 @@
             });
 
 
+
+
+
+            $('#id_admin_override_validation').addClass('addon-upload-failure-dependant')
```

Comment:
This is quite a large chunk of code. Does it all come from `devhub.js`? Are there any changes? Will it be missed there?